pkg/volume/empty_dir/empty_dir.go

```diff
@@ -275,7 +275,7 @@ func (ed *emptyDir) setupDir(dir string) error {
 
 	if fileinfo.Mode().Perm() != perm.Perm() {
 		// If the permissions on the created directory are wrong, the
-		// kubelet is probably running with a umask set.  In order to
+		// kubelet is probably running with an umask set.  In order to
```

Comment:
This is a funny one - in English I would say "a umask" because "umask" has an implied leading Y.  "you-mask".  Just like "user" or "utility", but unlike "uncle" or "underdog".